src/style.css

```diff
@@ -1 +1,83 @@
-/* Styles go here */
+.html {
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+}
+
+.page-body {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+}
+
+.logo {
+  left: 50px;
+  top: 10px;
```

Comment:
These styles are useless without positioning. If you remove them - nothing changes.